egs/swbd/s5c/RESULTS

```diff
@@ -138,6 +138,10 @@ exit 0
 %WER 12.1 | 1831 21395 | 89.1 7.1 3.8 1.3 12.1 48.1 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_12_0.0/eval2000_hires.ctm.swbd.filt.sys
 %WER 13.6 | 1831 21395 | 87.9 8.2 3.9 1.5 13.6 51.0 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
 
+# results with nnet3 cnn + tdnn.sh (20.2.2016) (2 epoch training on speed-perturbed and volume perturbed data)
```

Comment:
I am trying to ensure that we post results on all the subsets of HUB '00. Please post the results on callhm and swbd+callhm test sets.

In your case the improvements are not visible on the swbd subset. I remember that you had better results on the full set. 

(from your mail
%WER      swbd    callhm     overall
TDNN:        12.5      23.7         18.2
CNN+TDN   12.6     22.9          17.9)